src/Sentry.Unity.Editor/Native/SentryWindowsPlayer.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Sentry.Unity.Editor.Native
+{
+    internal static class SentryWindowsPlayer
+    {
+        internal static readonly ProcessStartInfo StartInfo = new()
+        {
+            UseShellExecute = false,
+            RedirectStandardOutput = true,
+            RedirectStandardError = true,
+            CreateNoWindow = true
+        };
+
+        internal static void AddNativeOptions(SentryUnityOptions options)
+        {
+        }
+
+        internal static void AddSentryToMain(SentryUnityOptions options)
+        {
+        }
+
+        internal static string LocateWindowsPlayerSource(IEditorApplication? editorApplication = null)
+        {
+            editorApplication ??= EditorApplicationAdapter.Instance;
+
+            var playerProjectPath = Path.Combine(editorApplication.ApplicationContentsPath, "PlaybackEngines", "windowsstandalonesupport", "source", "windowsplayer");
+            if (!Directory.Exists(playerProjectPath))
+            {
+                throw new DirectoryNotFoundException($"Failed to locate the WindowsPlayer source at {playerProjectPath}.");
+            }
+
+            return playerProjectPath;
+        }
+
+        internal static void CreateWindowsPlayerProject(string windowsPlayerSource, string windowsPlayerTarget, IDiagnosticLogger? logger)
+        {
+            EditorFileIO.CopyDirectory(windowsPlayerSource, windowsPlayerTarget, logger);
+
+            // TODO: Does the .props file have to look like that?
+            using var props = File.CreateText(Path.Combine(windowsPlayerTarget, "UnityCommon.props"));
+            props.Write(@"<?xml version=""1.0"" encoding=""utf-8""?>
+<Project xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
+</Project>");
+        }
+
+        internal static string LocateMSBuild(string vsWherePath, IDiagnosticLogger? logger)
+        {
+            StartInfo.FileName = vsWherePath;
+            StartInfo.Arguments = "-latest -requires Microsoft.Component.MSBuild -find MSBuild\\**\\Bin\\MSBuild.exe";
```

Comment:
How do we control what version we'll be working with here? If the user has multiple installed, some could be compatible and others not.